buku.py

```diff
@@ -4294,7 +4314,12 @@ def main():
     search_opted = True
     update_search_results = False
 
-    if args.sany is not None:
+    if args.stag is not None and args.sany is not None:
```

Comment:
We must check for the length too because users may pass `t ''` (empty string) to trick the program.